packages/block-library/src/media-text/edit.js

```diff
@@ -290,6 +298,28 @@ function MediaTextEdit( { attributes, isSelected, setAttributes } ) {
 		</PanelBody>
 	);
 
+	const dimensions = (
+		<PanelBody title={ __( 'Dimensions' ) }>
```

Comment:
The addition of the “Dimensions” panel could be problematic if padding or margin block support is adopted for this block as it also adds “Dimensions”. The cover block actually has this issue at the moment and there are a couple of PRs open addressing it.

- https://github.com/WordPress/gutenberg/pull/34102
- https://github.com/WordPress/gutenberg/pull/34065

There is also the proposed min-height block support that might reduce the changes needed here to toggling a support flag in the block.json plus the native changes.
- https://github.com/WordPress/gutenberg/pull/33970

Reply:
The best idea would then probably be to wait until these things are done?

Reply:
It might be easiest to hold fire until a decision is made on which direction we all wish to take at least. 

Whether we choose to inject the ad hoc control within the proposed dimensions panel slot (as per the Cover block) or go with the min-height block support, some changes will be needed. 

If there's no rush we can wait and only have to do the job once 🙂